dali/core/convert_cuda_test.cu

```diff
@@ -49,11 +58,16 @@ DEVICE_TEST(ConvertNormCUDA_Dev, float2int, 1, 1) {
   DEV_EXPECT_EQ(ConvertNorm<int8_t>(0.499f), 63);
   DEV_EXPECT_EQ(ConvertNorm<int8_t>(-1.0f), -127);
 
-
   DEV_EXPECT_EQ(ConvertNorm<uint16_t>(0.0f), 0);
   DEV_EXPECT_EQ(ConvertNorm<uint16_t>(1.0f), 0xffff);
   DEV_EXPECT_EQ(ConvertNorm<int16_t>(1.0f), 0x7fff);
   DEV_EXPECT_EQ(ConvertNorm<int16_t>(-1.0f), -0x7fff);
+  DEV_EXPECT_EQ(ConvertNorm<uint16_t>(0.0f), 0);
+
+  // float doesn't have appropriate precision, so we can only expect 23 MSBs to be valid.
+  DEV_EXPECT_GE(ConvertNorm<uint64_t>(1.0f),  0xffffff0000000000);
```

Comment:
how about using an epsilon for approximate comparison?

Reply:
There's no `DEV_EXPECT_NEAR`.